en/docs/integrations/build-your-first-integration.md

```diff
@@ -104,21 +104,21 @@ Follow this procedure to send an email with the summary of statistics to a speci
 3. Click **Save**.
 4. Click the last **+** icon in the low-code diagram.
 5. Click **API Calls** and then select **Mail by Choreo**.
-6. In the **Mail by Choreo Connection** window, enter details as follows: 
+6. In the **Mail by Choreo Connection** window, enter these details: 
 
-    1. In the **Recipient** field, enter the email address to which you want to send the email. For example, `"test@gmail.com"`
-    2. In the **Subject** field, enter `"Total COVID-19 Cases in the USA"`
+    1. In the **Recipient** field, enter the email address to which you want to send the email(for example, `"test@gmail.com"`).
```

Comment:
Should we just say E.g.?

Reply:
The style guide recommends to avoid abbreviations such as i.e., and e.g., [1].
[1] https://developers.google.com/style/abbreviations?hl=en#dont-use

Reply:
that's right.